Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -241,7 +244,12 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
 
     @objc
     private func didTapLearnMore() {
-        // didTapLearnMore will be enabled with task FXIOS-7427
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .shoppingLearnMoreReviewQualityButton)
+        guard let fakespotLearnMoreLink else { return }
+        tabManager.addTabsForURLs([fakespotLearnMoreLink], zombie: false, shouldSelectTab: true)
+        dismissViewController?()
     }
```

Comment:
Can we do this work in the view model. It's a bit too much knowledge for the view to have a tab manager and a link to redirect.